src/blas/impl/KokkosBlas2_team_gemv_spec.hpp

```diff
@@ -74,7 +74,6 @@ template <class TeamType, class MatrixType, class XVector, class YVector>
 struct TeamGEMV<TeamType, MatrixType, XVector, YVector, 0, false> {
   typedef typename Kokkos::Details::InnerProductSpaceTraits<
       typename MatrixType::non_const_value_type>::dot_type dot_type;
-  typedef typename XVector::non_const_value_type x_value_type;
```

Comment:
Good catch these unused definitions will probably bite us with more modern compilers that check more stringently for warnings.